test/integration/smoke/test_vm_life_cycle.py

```diff
@@ -120,6 +120,12 @@ def setUpClass(cls):
             mode=cls.services['mode']
         )
 
+        cls.cleanup = [
+            cls.virtual_machine,
+            cls.service_offering,
+            cls.account
+        ]
```

Comment:
a `cleanup` at cls level does not seem right. There is already a `cls._cleanup` and that should be used in this case.